internal/tsc_wrapped/compiler_host.ts

```diff
@@ -205,8 +206,9 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
     for (const root of this.options.rootDirs) {
       if (fileName.startsWith(root)) {
         // rootDirs are sorted longest-first, so short-circuit the iteration
-        // see tsconfig.ts.
-        return path.posix.relative(root, fileName);
+        // see tsconfig.ts. Since the fileName can already include multiple path segments with
+        // backslash delimiters, we need to manually convert the path to posix format.
```

Comment:
hmm, I'm surprised that fileName here could have backslashes, typescript should have canonicalized them already.
would you mind checking where that is coming from? it seems like it should have been converted somewhere else, this might be masking that bug

Also maybe it's time to have subtypes of `string` like `posixAbsolutePath` so the type checker could catch these, paths are such a mess :(

Reply:
Actually, it's my fault. I still ran an older version of the rules locally that didn't include https://github.com/bazelbuild/rules_typescript/commit/c52f257b6e33adc004ebdf0d573d3ec4e0c75c13. 

Commit https://github.com/bazelbuild/rules_typescript/commit/c52f257b6e33adc004ebdf0d573d3ec4e0c75c13 fixes the issue and there is no need to keep this PR open anymore. Thanks for the help.